.github/workflows/gitlint/action.yml

```diff
@@ -0,0 +1,42 @@
+# *******************************************************************************
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+# *******************************************************************************
+
+name: "Gitlint Action"
```

Comment:
What is the naming convention here? Other jobs have S-CORE CI - prepended. This one does not. Is that intentional?